.env.example

```
@@ -120,6 +120,12 @@ NEXT_PUBLIC_SENDGRID_SENDER_NAME=
 # Used for capturing exceptions and logging messages
 NEXT_PUBLIC_SENTRY_DSN=
 
+# Formbricks Experience Management Integration
+# [apiHost]+[environmentId]
+FORMBRICKS=
```

Not quite since is used on the server only. But I would go with something more descriptive.

```suggestion
FORMBRICKS_URL=
```